Database/Stored Procedures/Add_or_Update/add_or_update_visitor.sql

```diff
@@ -81,30 +125,58 @@ GO
 
 -- Test INSERT
 BEGIN TRAN
-EXEC sp_add_or_update_visitor NULL, 1, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 1
+EXEC sp_add_or_update_visitor NULL, 'yuriz@live.nl', 'Yuri', 'Vannisselroy', '0682006373', '1996-09-25', 'NULL', 'NULL', 'NULL', 'NULL', 'Nijmegen', 'Nederland', 1
+ROLLBACK TRAN
+GO
+
+BEGIN TRAN
```

Comment:
(0 rows affected)
Msg 515, Level 16, State 2, Procedure sp_add_or_update_country, Line 10 [Batch Start Line 134]
Cannot insert the value NULL into column 'name', table 'FestiBase.dbo.COUNTRY'; column does not allow nulls. INSERT fails.


Deze test zou moeten lukken..  Maar hij geeft een error.